src/app/modules/vault-filter/components/organization-name-badge.component.html

```diff
@@ -0,0 +1,9 @@
+<button
+  bit-badge
+  [style.color]="textColor"
```

Comment:
Just wondering - did you try creating a component scoped stylesheet before doing this inline?

I've tried in the past, and it never gets included in compilation and I have no clue why 😵‍💫

Reply:
Maybe it can be one with ngClass 🤔